layersvt/screenshot.cpp

```diff
@@ -92,6 +93,7 @@ static inline void local_free_getenv(const char *val) {}
 
 const char *env_var_old = "_VK_SCREENSHOT";
 const char *env_var = "VK_SCREENSHOT_FRAMES";
+const char *env_var_format = "VK_SCREENSHOT_FORMAT";
```

Comment:
These three lines are duplicated. Can #ifdef's be rearranged so that they occur only once?